packages/components/src/__future__/Select/Select.spec.tsx

```diff
@@ -356,4 +356,83 @@ describe("<Select />", () => {
       })
     })
   })
+
+  describe("Popover portal", () => {
+    it("has accessible trigger controls", async () => {
+      render(<SelectWrapper id="id--select" isOpen />)
```

Comment:
No need to declare the id in these tests

Reply:
And they're outa here ⚾ 🌬️ Nice catch, some left overs of when I had some janky selectors in there